cmd/zoekt-indexserver/config.go

```diff
@@ -50,6 +50,7 @@ type ConfigEntry struct {
 	ExcludeTopics          []string
 	Active                 bool
 	NoArchived             bool
+	RepoNameFormat         string
```

Comment:
Maybe prefix name with `Gerrit` since it is gerrit specific?

Reply:
Done